tests/hooks/conftest.py

```
@@ -185,48 +207,59 @@ def aap_mtv_settings(
+            )
+
+            forklift_controller = ForkliftController(
+                client=ocp_admin_client,
+                name=FORKLIFT_CONTROLLER_NAME,
+                namespace=mtv_namespace,
+                ensure_exists=True,
+            )
+
+            LOGGER.info(
+                f"Patching ForkliftController with aap_url='{awx_deployment}', "
+                f"aap_token_secret_name='{token_secret_name}', and aap_insecure_skip_verify=true"
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)